ci/tasks/create-notification-snippets.yml

```diff
@@ -41,6 +42,13 @@ run:
       echo "- <https://github.com/alphagov/pay-nginx-proxy/releases/tag/alpha_release-${NGINX_RELEASE_NUMBER}|nginx-proxy v${NGINX_IMAGE_TAG}>" | \
       tee -a snippet/failure || true
 
+      # Disable the shellcheck rull telling us this isn't an if-then-else. We know, and it's the desirable
```

Comment:
"rule"

Reply:
Ha good spot, this same comment is copied loads of times in this file 😆 

I'll update the misspelling in the follow on PR for staging